easybuild/easyconfigs/n/Nextflow/Nextflow-22.10.1.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'Binary'
+
+name = 'Nextflow'
+version = '22.10.1'
+
+homepage = 'https://www.%(namelower)s.io/'
+description = """Nextflow is a reactive workflow framework and a programming DSL 
+ that eases writing computational pipelines with complex data"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/%(namelower)s-io/%(namelower)s/releases/download/v%(version)s/']
+sources = ['%(namelower)s-%(version)s-all']
+checksums = ['c5edc9fc81d9c6eb4f628505f14f8bb4fe044d4fc7d9ef60654ba0d73abf5a80']
+
+dependencies = [('Java', '11', '', True)]
+
+install_cmd = "[ -d %(builddir)s/bin ] || mkdir -p %(installdir)s/bin &&"
```

Comment:
Why the `-d` check, `mkdir -p` will work just fine if the directory already exists?